Bio/GenBank/Scanner.py

```diff
@@ -476,16 +477,16 @@ def parse_records(self, handle, do_features=True):
         """
         # This is a generator function
         while True:
-            record = self.parse(handle, do_features)
-            if record is None:
+            next_record = self.parse(handle, do_features)
```

Comment:
I don't see a problem with reusing ``record`` here, it is short and consise. Perhaps my other programming language background blinds me to the shadowing problem - I am still sometimes caught out by Python's scoping rules.